extension/src/panel.ts

```diff
@@ -80,18 +80,13 @@ export class HistoryDebuggerPanel {
     private readonly secretStorage: vscode.SecretStorage,
     protected readonly server: Server,
   ) {
-    const column = vscode.window.activeTextEditor ? vscode.window.activeTextEditor.viewColumn : undefined
-    this.panel = vscode.window.createWebviewPanel(
-      HistoryDebuggerPanel.viewType,
-      "Temporal",
-      column || vscode.ViewColumn.Beside,
-      {
-        // Enable javascript in the webview
-        enableScripts: true,
-        // And restrict the webview to only loading content from our extension's compiled directory.
-        localResourceRoots: [vscode.Uri.joinPath(extensionUri, "webview/dist")],
-      },
-    )
+    this.panel = vscode.window.createWebviewPanel(HistoryDebuggerPanel.viewType, "Temporal", vscode.ViewColumn.Beside, {
+      // Enable javascript in the webview
+      enableScripts: true,
+      // And restrict the webview to only loading content from our extension's compiled directory.
+      localResourceRoots: [vscode.Uri.joinPath(extensionUri, "webview/dist")],
+      retainContextWhenHidden: true,
```

Comment:
I was finally able to reproduce the bug where the panel data would disappear (it happens when the file with the debugger breakpoint and the webview are in the same panel). To fix it I've set `retainContextWhenHidden` to `true`. 

I've also removed the column check so that we can move the `instance.show()` into an else block on install and the panel will always show up beside.